data/card_catalogue.json

```diff
@@ -628,87 +670,87 @@
         {
           "title": "Whip",
           "upgrade": "Pitch",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Throw an object up to 10m",
+            "rules": "Throw an object up to 10 squares",
             "flavour": "\"Real Good\" -Divo",
             "image": "assets/core/card/illustration/Whip.png"
           }
         },
         {
           "title": "Pitch",
           "upgrade": "Hurl",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "Throw an object up to 50m",
+            "rules": "Throw an object up to 50 squares",
             "flavour": "\"I will not rest until every window is smashed. They are an insult to me and my kind\" -Little Jimmy, neighborhood rascal",
             "image": "assets/core/card/illustration/Pitch.png"
           }
         },
         {
           "title": "Hurl",
           "upgrade": "Launch",
-          "tier": "green",
+          "tier": "red",
           "details": {
-            "rules": "Throw an object up to 300m",
+            "rules": "Throw an object as far as you want.",
             "flavour": "I want this thing gone, understand? I want to never see it again.",
             "image": "assets/core/card/illustration/Hurl.png"
           }
         },
         {
           "title": "Launch",
           "upgrade": "",
-          "tier": "red",
+          "tier": "gold",
           "details": {
-            "rules": "Throw an object as far as you want",
+            "rules": "Throw an object as far as you want. It cannot be stopped.",
             "flavour": "I guess in a way, we're all still chasing it.",
             "image": "assets/core/card/illustration/Launch.png"
           }
         },
         {
           "title": "Flirt",
           "upgrade": "Charm",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "A compliment increases your rapport with an NPC",
+            "rules": "Increases your rapport with a visible character",
             "flavour": "Are you related to Jean-Claude Van Damme? Because Jean-Claude Van Damme you’re sexy!",
             "image": "assets/core/card/illustration/Flirt.png"
           }
         },
         {
           "title": "Charm",
           "upgrade": "Arouse",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "Causes an NPC to like you.",
+            "rules": "Causes a visible character to like you.",
```

Comment:
Adding the visible requirement, the ability to just target anyone in existence should be mostly a gold level thing